src/Compilers/VisualBasic/Portable/Symbols/SymbolExtensions.vb

```diff
@@ -419,8 +419,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return origMember
             End If
 
-            Dim substituted = DirectCast(type, SubstitutedNamedType)
-            Return DirectCast(substituted.GetMemberForDefinition(origMember), T)
+            Dim substituted = TryCast(type, SubstitutedNamedType)
+            If substituted IsNot Nothing Then
+                Return DirectCast(substituted.GetMemberForDefinition(origMember), T)
+            End If
+
+            'Dim errorType = DirectCast(type, SubstitutedErrorType)
```

Comment:
What's going on here?

Reply:
That's a WIP commit (to transfer from work to home).